apm-webapp/src/main/java/org/apache/skywalking/oap/server/webapp/ApplicationStartUp.java

```diff
@@ -68,6 +68,7 @@ public static void main(String[] args) throws Exception {
             .builder()
             .port(port, SessionProtocol.HTTP)
             .service("/graphql", new OapProxyService(oapServices))
+            .service("/debugging/config/dump", new OapProxyService(oapServices))
```

Comment:
I think we have `/status/*`

ref to 
- https://skywalking.apache.org/docs/main/next/en/status/query_ttl_setup/
- https://skywalking.apache.org/docs/main/next/en/status/query_cluster_nodes/